### PR TITLE
Max proof read

### PR DESCRIPTION
**Pull request by kain88-de**
A collection of changes after reading through the whole paper. I found some spelling errors and changed text where I thought it helps readability.

### CODE REVIEW

File: papers/oliver_beckstein/oliver_beckstein.rst [hunk 1]
```diff
@@ -132,9 +133,9 @@ MDAnalysis currently supports more than 25 different file formats and covers the
...
 Since the original publication :cite:`Michaud-Agrawal:2011fu`, improvements in speed and data structures make it now possible to work with terabyte-sized trajectories containing up to ~10 million particles.
-MDAnalysis also comes with specialized analysis classes in the MDAnalysis.analysis module that are unique to MDAnalysis such as the LeafletFinder graph-based algorithm for the analysis of lipid bilayers :cite:`Michaud-Agrawal:2011fu` or the Path Similarity Analysis for the quantitative comparison of macromolecular conformational changes :cite:`Seyler:2015fk`.
+MDAnalysis also comes with specialized analysis classes in the MDAnalysis.analysis module that are unique to MDAnalysis such as LeafletFinder a graph-based algorithm for the analysis of lipid bilayers :cite:`Michaud-Agrawal:2011fu` or Path Similarity Analysis for the quantitative comparison of macromolecular conformational changes :cite:`Seyler:2015fk`.
```
Comment (orbeckst) [inline papers/oliver_beckstein/oliver_beckstein.rst:138]: LeafletFinder, a ....: cite:`Michaud-Agrawal:2011fu`, ...

**Comment by orbeckst** [comment]
@dotsdl unless you are dying to review & merge I will do it. I am happy.

**Comment by dotsdl** [comment]
Not dying; have enough to do. Have at it!

**Comment by orbeckst** [comment]
Haha. Will do.

**Pull request merged**